process-automation/latest/model/connectors/docusign.md

```diff
@@ -98,6 +98,57 @@ When a Process Automation process is instantiated this way, the following variab
 | envelopeId | string | Envelope ID of the document. |
 | documents | array | Documents related to the envelope and data related to them like uri, and id. |
 
```

Comment:
The following internal page link is not working: 

`* [DOWNLOAD_DOCUMENT](#download_document)`